crates/napi/src/threadsafe_function.rs

```diff
@@ -494,9 +494,7 @@ impl<T: 'static> ThreadsafeFunction<T, ErrorStrategy::CalleeHandled> {
                 callback: Box::new(move |d: Result<JsUnknown>| {
                   sender
                     .send(d.and_then(|d| D::from_napi_value(d.0.env, d.0.value)))
-                    .map_err(|_| {
-                      crate::Error::from_reason("Failed to send return value to tokio sender")
-                    })
+                    .or(Ok(()))
```

Comment:
Maybe it's worth to add comments here to explain that it's okay to ignore the `send error`

Reply:
On it

Reply:
Done - mind tagging once merged?